src/test/java/com/commercetools/sync/producttypes/utils/producttuypeactionutils/BuildAttributeDefinitionUpdateActionsTest.java

```diff
@@ -1,14 +1,14 @@
 package com.commercetools.sync.producttypes.utils.producttuypeactionutils;
 
 import com.commercetools.sync.commons.exceptions.BuildUpdateActionException;
-import com.commercetools.sync.commons.exceptions.DifferentTypeException;
 import com.commercetools.sync.commons.exceptions.DuplicateNameException;
 import com.commercetools.sync.producttypes.ProductTypeSyncOptions;
 import com.commercetools.sync.producttypes.ProductTypeSyncOptionsBuilder;
 import io.sphere.sdk.client.SphereClient;
 import io.sphere.sdk.commands.UpdateAction;
 import io.sphere.sdk.products.attributes.AttributeDefinition;
 import io.sphere.sdk.products.attributes.AttributeDefinitionBuilder;
+import io.sphere.sdk.products.attributes.LocalizedStringAttributeType;
```

Comment:
There is a type in the name of the package of this test class:
`producttuypeactionutils` -> `producttypeactionutils` ;)

Reply:
83a625c638edb93103fca429e43f00bced3cdde0